publish/publish_module.gradle

```diff
@@ -0,0 +1,97 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+if (versions == null || versions.publishVersion == null) {
+    throw new IllegalStateException("Unable to reference publishVersion")
+}
+
+task androidSourcesJar(type: Jar) {
+    archiveClassifier.set('sources')
+    if (project.plugins.findPlugin("com.android.library")) {
+        // For Android libraries
+        from android.sourceSets.main.java.srcDirs
+        from android.sourceSets.main.kotlin.srcDirs
+    } else {
+        // For pure Kotlin libraries
+        from sourceSets.main.java.srcDirs
+        from sourceSets.main.kotlin.srcDirs
+    }
+}
+
+artifacts {
+    archives androidSourcesJar
+}
+
+group = versions.publishGroupID
+version = versions.publishVersion
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+
+                groupId versions.publishGroupID
+                artifactId versions.publishArtifactID
+                version versions.publishVersion
+
+                if (project.plugins.findPlugin("com.android.library")) {
+                    from components.release
+                } else {
+                    artifact("$buildDir/libs/${project.getName()}-${version}.jar")
+                }
+
+                artifact androidSourcesJar
+
+                pom {
+                    name = versions.publishArtifactID
+                    description = 'A library to dynamically format your EditTexts to take currency inputs.'
+                    url = 'https://github.com/CottaCush/CurrencyEditText'
+                    licenses {
+                        license {
+                            name = 'Apache License'
+                            url = 'https://github.com/CottaCush/CurrencyEditText/blob/master/LICENSE.md'
+                        }
+                    }
+                    developers {
+                        developer {
```

Comment:
Do we need to hardcode contributor names?
Is it possible for it to be pulled from the commit history?
Or do we even need to add the information at all?

Reply:
Hmm. On being pulled from commit history, I can check if that's possible. 
On whether the information actually needs to be added, I don't think its necessary. 
Do you think we shouldn't add it if it isn't absolutely necessary?

Reply:
Don't add it. I mean we have other contributors as well.

Reply:
Alright. Sounds good.